src/CommunityVoices/App/Api/Component/FileProcessor.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+ namespace CommunityVoices\App\Api\Component;
+ use Symfony\Component\HttpFoundation;
+
+ class FileProcessor {
+     const ERR_NO_ATTRIBUTIONS = 'The source table must provide an "attribution" column';
```

Comment:
For a generic file processor, all the constants here seem rather specific to batch upload.  Could they be stored elsewhere and passed in?

Reply:
Given that the batch function is here, I can see this being present.  But I could also see batch processor being an entirely separate file given how much it dominates this one.

Reply:
@Sammidysam Good points. Both of these can be fixed.